tts/UI.def.lua

```diff
@@ -96,12 +98,11 @@ UI = {}
 ---@field attributes nil | tts__UIElementBase_Attributes
 ---@field children nil | tts__UIElement[]
 
----@shape tts__UIInputElementBase : tts__UIElementBase
```

Comment:
Whilst there is indeed an error in the existing types, `tts__UIInputElementBase` should still exist. It's just that it should override the `attributes` field with a new `tts__UIInputElementBase_Attributes` (which itself extends `tts__UIElementBase_Attributes`).

The logic here is that input elements are their own class of elements as they all exhibit common behaviour, namely have an `onValueChanged` callback. Sometimes you may wish to write code that accepts any input element and attaches a `onValueChanged` callback etc.